packages/kbn-apm-synthtrace/src/lib/infra/infra_synthtrace_es_client.ts

```diff
@@ -30,13 +34,26 @@ export class InfraSynthtraceEsClient extends SynthtraceEsClient<InfraDocument> {
       'metrics-aws*',
     ];
   }
+
+  getDefaultPipeline(
+    {
+      includeSerialization,
+    }: {
+      includeSerialization?: boolean;
+    } = { includeSerialization: true }
+  ) {
+    return infraPipeline({ includeSerialization });
+  }
 }
 
-function infraPipeline() {
+function infraPipeline({ includeSerialization }: Pipeline = { includeSerialization: true }) {
   return (base: Readable) => {
+    const serializationTransform = includeSerialization ? [getSerializeTransform()] : [];
+
```

Comment:
This modification has been implemented in the same way as in other pipelines, such as:
- APM https://github.com/elastic/kibana/blob/59d3ac65ec59408f22c31e629ea563292943727e/packages/kbn-apm-synthtrace/src/lib/apm/client/apm_synthtrace_es_client/apm_pipeline.ts#L50-L71

- Entities https://github.com/elastic/kibana/blob/d560c490958d53681bf0d7cfeaff3fe62b617c1a/packages/kbn-apm-synthtrace/src/lib/entities/entities_synthtrace_es_client.ts#L38-L56